x/ecocredit/base/keeper/features/msg_create_batch.feature

```diff
@@ -18,40 +18,40 @@ Feature: Msg/CreateBatch
     Background:
       Given a credit type with abbreviation "C"
       And a credit class with class id "C01" and issuer alice
-      And a project with project id "C01-001"
+      And a project with project id "P001" enrolled in "C01"
```

Comment:
There is no scenario to cover issuing a credit batch for a project that does not have an `ACCEPTED` enrollment status. Maybe this Rule could be expanded: "The project must exist and have an accepted enrollment to a credit class"

Reply:
Added a test to cover this: https://github.com/regen-network/regen-ledger/pull/2167/commits/4336def78e798ac87a8fe61b63a31103e217cb50

Reply:
Looks good